### PR TITLE
feat: Add a transition message

### PR DESCRIPTION
**Pull request by simar7**
Fixes: https://github.com/aquasecurity/tfsec/issues/2066

### CODE REVIEW

**Review by itaysk — APPROVED** [review]

**Comment by kawikao** [comment]
This is breaking the tfsec-sarif-action upload because it's not valid sarif.

File: cmd/tfsec/main.go [hunk 1]
```diff
@@ -8,7 +8,21 @@ import (
...
+for the time being, although our engineering 
+attention will be directed at Trivy going forward.
+
+You can read more here: 
+https://github.com/aquasecurity/tfsec/discussions/1994
+======================================================
+`
+
 func main() {
+	fmt.Print(transitionMsg)
```
Comment (yu-iskw) [inline cmd/tfsec/main.go:25]: @simar7 @itaysk Can we print it out as standard error? The standard out message disable us to parse the result of JSON format.

Comment (yu-iskw) [reply cmd/tfsec/main.go:25]: I opened a pull request. Let's discuss that there. 

https://github.com/aquasecurity/tfsec/pull/2099

Comment (itaysk) [reply cmd/tfsec/main.go:25]: you're right, sorry. It shouldn't be printed in machine-readable formats like json, csv, junit, sarif, etc.

**Pull request merged**